test/functional/p2p_invalid_messages.py

```diff
@@ -3,13 +3,18 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test node responses to invalid network messages."""
+import struct
```

Comment:
nit: empty line between the docstring and the first import

Reply:
will be good to add the rule to the python-lint script.